solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/AddReplicaPayload.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.client.solrj.request.beans;
+
+import org.apache.solr.common.annotation.JsonProperty;
+import org.apache.solr.common.util.ReflectMapWriter;
+
+import java.util.List;
+import java.util.Map;
+
+public class AddReplicaPayload implements ReflectMapWriter {
+  @JsonProperty
+  public String shard;
+
+  @JsonProperty
+  public String _route_;
+
+  // TODO Could 'node' be replaced (in v2 and v1) by just specifying a 'createNodeSet' of size 1?
+  @JsonProperty
+  public String node;
+
+  // TODO Should this just be 'nodeSet' to match the name used by create-shard and other APIs?
```

Comment:
LIkewise seems like a good idea.   Would these be done in this PR, or in future tickets?

Reply:
I think a separate ticket, since there's technically deprecation and other concerns that come up here.  Maybe I can handle it as a part of SOLR-15542, which I created based on your comment above?

Reply:
I like doing it as part of SOLR-15542 as well.